src/test/java/com/griddynamics/testCollectionTask/TestCollectionTask.java

```diff
@@ -0,0 +1,82 @@
+package com.griddynamics.testCollectionTask;
+
+import com.griddynamics.collectionsTask.CollectionsTask.FileFormatException;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static com.griddynamics.collectionsTask.CollectionsTask.*;
+import static org.testng.AssertJUnit.assertEquals;
+
+public class TestCollectionTask {
+    List<Car> cars = new ArrayList<>();
+    List<Truck> trucks = new ArrayList<>();
+    List<Vehicle> vehicles = new ArrayList<>();
+
+    @BeforeMethod
+    public void init() {
+        cars.clear();
+        trucks.clear();
+        vehicles.clear();
+    }
+
+    @Test
+    public void testUsual() throws FileFormatException, IOException {
+        String filePath = getFilePath("vehicle_data_collections.txt");
+        List<Vehicle> rightVehiclesOrder = new ArrayList<>(Arrays.asList(new Car("VAZ-2000", "VAZ", 50, "pupu"),
+                new Car("VAZ-2101", "VAZ", 40, "sedan"),
+                new Truck("KamAZ-3011", "AZ", 2, 900),
+                new Truck("KamAZ-4211", "KamAZ", 5, 8000)));
+        read(cars, trucks, filePath);
+        merge(cars, trucks, vehicles);
+        sortListOfVehicles(vehicles);
+        for (int i = 0, vehiclesSize = vehicles.size(); i < vehiclesSize; i++) {
```

Comment:
before checking each of the elements you can quickly check that actual list's size is 4 and if it is correct proceed with other checks